### PR TITLE
002 fixing docker

### PR DESCRIPTION
**Pull request by PhilipGardinerK**
Throws an error because it tries to do the "getAllDatabses" endpoint and fails

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -6,9 +6,9 @@ COPY package*.json ./
 
 ARG API_URL
 
-ENV API_URL ${API_URL}
+ENV API_URL = "https://ivpztrmt3p.eu-west-1.awsapprunner.com/swagger#/Test%20API/testConnection"
```
Comment (shaunganley) [inline Dockerfile]: nope, don't hardcode this and the API URL doesn't need the swagger path etc

**Review by shaunganley — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (PhilipGardinerK) [reply Dockerfile]: ok will change that now

Comment (PhilipGardinerK) [reply Dockerfile]: Thats it changed

File: Dockerfile [hunk 2]
```diff
@@ -6,9 +6,9 @@ COPY package*.json ./
 
 ARG API_URL
 
-ENV API_URL ${API_URL}
+ENV API_URL = ${API_URL}
```
Comment (shaunganley) [inline Dockerfile]: I don't think the = is correct syntax based on what it was before

**Review by shaunganley — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 2 above)
Comment (PhilipGardinerK) [reply Dockerfile]: Yeah just removed the url and the equals sign :( thats it fixed

**Review by shaunganley — APPROVED** [review]

**Review by ThomasS-kainos — APPROVED** [review]

**Pull request merged**